### PR TITLE
tensor math.py func

### LINKED ISSUE #21203: add_n

### LINKED ISSUE #21204: digamma

### LINKED ISSUE #21205: erfinv

### PR DESCRIPTION
**Pull request by TangoIndiaMango**
Close #21205
Close #21204
Close #21203

### CODE REVIEW

File: ivy/functional/frontends/paddle/tensor/math.py [hunk 1]
```diff
@@ -410,10 +410,26 @@ def any(x, axis=None, keepdim=False, name=None):
...
+
+@with_unsupported_dtypes({"2.5.0 and below": "bool"}, "paddle")
+@to_ivy_arrays_and_back
+def erfinv(x, name=None):
+    return ivy.erfinv(x)
+
 def diff(x, n=1, axis=-1, prepend=None, append=None, name=None):
     return ivy.diff(x, n=n, axis=axis, prepend=prepend, append=append)
 
-
```
Comment (KareemMAX) [inline ivy/functional/frontends/paddle/tensor/math.py:416]: Seems like you forgot to revert this change, which breaks our linting rules

**Review by KareemMAX — CHANGES_REQUESTED** [review]
Hey @TangoIndiaMango,

Seems like you forgot to add unit tests. also both `add_n` and `erfinv` doesn't seem like they exist in Ivy for now. Are you sure this code is runnable?

Also to streamline the review process. Can you make a comment with "add_frontend_checklist" (with no quotes nor any spaces or new lines)? When you do that a checklist will appear which we use to make sure that your PR is good to merge. Please tick anything in it that you feel you have completed

**Review by KareemMAX — CHANGES_REQUESTED** [review]
I guess you removed `diff()` in the process of updating your PR, please revert that back.

Also the tests are still missing, please refer to our [open tasks guide](https://unify.ai/docs/ivy/overview/contributing/open_tasks.html#frontend-apis)

**Comment by TangoIndiaMango** [comment]
add_frontend_checklist

**Pull request closed without merging**